<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Terminallog</class>
 <widget class="QWidget" name="Terminallog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Terminallog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Ausgabe von:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="progName">
       <property name="text">
        <string>Process</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="logView">
     <property name="undoRedoEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="killButton">
       <property name="text">
        <string>Prozess &amp;killen</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="state">
       <property name="text">
        <string>Prozessstatus</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="toolTip">
        <string>solange das Programm läuft, lässt sich das Schließen nur erzwingen, indem das Fenster geschlossen wird.</string>
       </property>
       <property name="text">
        <string>Fenster &amp;Schließen</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
